apps/marsha/templates/services/app/svc_xapi.yml.j2

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: marsha
+    service: marsha
+    version: "{{ marsha_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: marsha-xapi-{{ deployment_stamp }}  # name of the service should be host name in nginx
+  namespace: "{{ project_name }}"
+spec:
+  ports:
+  - name: 8000-tcp
+    port: 8000
+    protocol: TCP
+    targetPort: 8000
```

Comment:
Use `{{ marsha_django_port }}` instead of the hardcoded 8000 ?
It was already like this before your PR in the service definition, but I think it's an opportunity to improve this.